src/columns/nano-id.ts

```diff
@@ -0,0 +1,24 @@
+import * as glide from "../glide";
+import { nanoid, customAlphabet } from "../../node_modules/nanoid/async";
+
+export default glide
+    .columnNamed("Nano Id")
+    .withCategory("Text")
+    .withDescription(`A tiny (108 bytes), secure, URL-friendly, unique string ID generator for JavaScript`)
+    .withNumberParam("num", "Length")
+    .withStringParam("alphabet", "Custom Alphabet")
+    .withAuthor("Andrey Sitnik", "https://github.com/ai/nanoid")
+    .withFailingTest({ num: 10 }, "test")
+    .withAbout(
+        `
+        A tiny, secure, URL-friendly, unique string ID generator for JavaScript.`
+    )
+    .withStringResult()
+    .run(async ({ alphabet, num }) => {
+        if (num !== undefined && alphabet !== undefined) {                        
+            const nanoid = customAlphabet(alphabet, num);
+            return await nanoid();
```

Comment:
I expect this `nanoid` function takes a `seed` parameter.

Reply:
@dvdsgl it does not. See this: https://github.com/ai/nanoid#async

It does offer a [Custom Random Bytes generator](https://github.com/ai/nanoid#custom-random-bytes-generator) that does take a seed. I can implement that too.

Reply:
@IanLeatherbury yes – we need that!